vmprof/reader.py

```diff
@@ -1,23 +1,51 @@
 from __future__ import print_function
 import re
 import struct
+import array
```

Comment:
Original code sorts standard libs in ascii order.
Keep it sorted if no special reason to break it.